app/views/producers/_fat.html.haml

```diff
@@ -1,7 +1,7 @@
 .row.active_table_row{"ng-if" => "open()", "ng-click" => "toggle($event)", "ng-class" => "{'open' : open()}"}
   .columns.small-12.fat.text-center{"ng-show" => "open() && shopfront_loading"}
     %p.fullwidth
-      %img.spinner.text-center{ src: image_path("spinning-circles.svg") }
```

Comment:
won't the lack of `.text-center` affect the style of the component?

Reply:
AFAIK no, because the parent has the same property : .text-center
Also, I've tested and it seems to be ok.